test/e2e/fixture-builder.js

```diff
@@ -845,20 +845,14 @@ class FixtureBuilder {
     });
   }
 
-  withPreferencesControllerSmartTransactionsOptedIn() {
-    return this.withPreferencesController({
-      preferences: {
-        smartTransactionsOptInStatus: true,
-        tokenNetworkFilter: {},
-      },
-    });
-  }
-
+  /**
+   * @deprecated this method should not be used, as the `smartTransactionsOptInStatus` value is overridden by the migration 135
+   * Use the step by step flow to disable this setting.
+   */
   withPreferencesControllerSmartTransactionsOptedOut() {
```

Comment:
:information_source: leaving the method with the `@deprecated` info, so this is not used.
I could have deleted, but I'm afraid this can be re-introduced again, so leaving this here for a while before removing it completely.
Side: removed the `tokenNetworkFilter` as it's irrelevant here